.github/workflows/github-teams-check-existance.yml

```diff
@@ -0,0 +1,173 @@
+name: Github Team - Check Existance
+
+on:
+  # Runs on schedule
+  schedule:
+    - cron: "0 10 * * 1-5" # At 10:00 AM, Monday through Friday
```

Comment:
10 AM, according to which time zone? My recommendation would be to run it within work ours for both EU and US West coast, so that we can all react if something gets triggers/goes off/etc.- e.g., 7 AM PST (3 PM GMT).

Reply:
Seems to be GMT currently. I will update this to be 3pm. There is a feature request in GH for Timezone support around cron jobs similar to the GitLab experience